src/main/java/gtPlusPlus/recipes/CokeAndPyrolyseOven.java

```diff
@@ -54,89 +53,95 @@ public static void postInit() {
+            new ItemStack(ModItems.itemSugarCharcoal, 20),
+            new ItemStack(ModItems.itemSugarCoke, 20));
 
-            GTValues.RA.stdBuilder()
-                .itemInputs(aInputs2[i], GTUtility.getIntegratedCircuit(4))
-                .itemOutputs(aOutputs[i])
-                .fluidInputs(FluidUtils.getFluidStack("nitrogen", 1000))
-                .fluidOutputs(FluidUtils.getFluidStack("charcoal_byproducts", 2000))
-                .eut(16)
-                .duration(25 * SECONDS)
-                .addTo(pyrolyseRecipes);
-            // Coke to Wood tar/Wood gas
-            GTValues.RA.stdBuilder()
-                .itemInputs(aOutputs[i], GTUtility.getIntegratedCircuit(5))
-                .itemOutputs(Materials.Ash.getDust(5))
-                .fluidInputs(FluidUtils.getFluidStack("steam", 2000))
-                .fluidOutputs(Materials.WoodTar.getFluid(4000))
-                .eut(240)
-                .duration(75 * SECONDS)
-                .addTo(pyrolyseRecipes);
-
-            GTValues.RA.stdBuilder()
-                .itemInputs(aOutputs[i], GTUtility.getIntegratedCircuit(6))
-                .itemOutputs(Materials.Ash.getDust(5))
-                .fluidInputs(FluidUtils.getFluidStack("steam", 2000))
-                .fluidOutputs(Materials.WoodGas.getGas(6000))
-                .eut(240)
-                .duration(75 * SECONDS)
-                .addTo(pyrolyseRecipes);
-        }
         // Coke & Coal
         GTValues.RA.stdBuilder()
             .itemInputs(
-                GTOreDictUnificator.get(OrePrefixes.gem, Materials.Coal, 12L),
-                ItemUtils.getItemStackOfAmountFromOreDict("fuelCoke", 6))
-            .itemOutputs(ItemUtils.getItemStackOfAmountFromOreDict("fuelCoke", 14))
+                GTOreDictUnificator.get(OrePrefixes.gem, Materials.Coal, 12),
+                GTOreDictUnificator.get("fuelCoke", 6))
+            .itemOutputs(GTOreDictUnificator.get("fuelCoke", 14))
             .fluidInputs(GTModHandler.getSteam(2000))
-            .fluidOutputs(FluidUtils.getFluidStack("fluid.coalgas", 5040))
+            .fluidOutputs(new FluidStack(CoalTar.Coal_Gas, 5040))
             .eut(240)
             .duration(1 * MINUTES)
             .addTo(cokeOvenRecipes);
     }
 
+    private static void addCokingRecipes(ItemStack plant, ItemStack charcoal, ItemStack coke) {
+        // Plant to Charcoal
+        GTValues.RA.stdBuilder()
+            .itemInputs(plant, GTUtility.getIntegratedCircuit(3))
+            .itemOutputs(charcoal)
+            .fluidOutputs(Materials.Creosote.getFluid(2000))
+            .eut(16)
+            .duration(25 * SECONDS)
+            .addTo(pyrolyseRecipes);
+
+        GTValues.RA.stdBuilder()
+            .itemInputs(plant, GTUtility.getIntegratedCircuit(4))
+            .itemOutputs(charcoal)
+            .fluidInputs(Materials.Nitrogen.getFluid(2000))
+            .fluidOutputs(Materials.CharcoalByproducts.getFluid(4000))
```

Comment:
```suggestion
            .fluidInputs(Materials.Nitrogen.getGas(2000))
            .fluidOutputs(Materials.CharcoalByproducts.getGas(4000))
```

getFluid doesn't actually return the fluids.